flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -408,9 +300,10 @@ static int32_t vtol_follower_control_simple(const float dT,
  * Takes in @ref PositionActual and compares it to @ref PositionDesired
  * and computes @ref VelocityDesired
  */
-int32_t vtol_follower_control_endpoint(const float dT, const float *hold_pos_ned)
+int32_t vtol_follower_control_endpoint(const float dT, const float *hold_pos_ned,
```

Comment:
actually this change wasn't needed. vtol_follower_control_endpoint never is _not_ passed NULL for the last parameter